pgbot/__init__.py

```diff
@@ -299,6 +299,11 @@ async def handle_message(msg: discord.Message):
             await embed_utils.send(
                 common.entries_discussion_channel, title, "", color, fields=fields
             )
+        elif (
+            random.random() < (await emotion.get("depression"))["value"] / 200
```

Comment:
hmm, this `(await emotion.get("depression"))` can also return an int, are we handling that case correctly here. Maybe `emotion.get` and `emotion.update` functions need some modifications to support this depression emotion

Reply:
Okay, I added ze handling of the case for `emotion.get()`, but as I already have a custom function for depression, I didn't add the handling for `emotion.update()`